apps/web/components/availability/v2/Schedule.tsx

```diff
@@ -0,0 +1,406 @@
+import { useTranslation } from "next-i18next";
```

Comment:
This is a new file for new schedule component. Other should be reverted and deprecated when required.

Reply:
FYI the same file exists in packages/ui/v2/modules/availability - it may not have existed there yet at the time you created this file.